app/controllers/course/enrol_requests_controller.rb

```diff
@@ -0,0 +1,132 @@
+class Course::EnrolRequestsController < Course::ModuleController
+  load_and_authorize_resource through: :course
+
+  def index #:nodoc:
+    @staff_requests = @enrol_requests.staff
+    @student_requests = @enrol_requests.student
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new #:nodoc:
```

Comment:
Assignment Branch Condition size for new is too high. [30.48/15]<br>Method has too many lines. [26/10]